### PR TITLE
add Odin support

### PR DESCRIPTION
**Pull request by spsandwichman**

### CODE REVIEW

File: gengo/languages.yaml [hunk 1]
```diff
@@ -446,6 +446,14 @@ OCaml:
...
       - ml
       - mli
+Odin:
+  category: programming
+  color: "#3882D2"
+  matchers:
+    extensions:
+      - odin
+    interpreters:
+      - odin
```
Comment (spenserblack) [inline gengo/languages.yaml:456]: Can Odin be used as an interpreter? Some quick searching didn't turn anything substantial up. This entry is used for executable text files in Unix-like systems, where a file extension might not exist. For example:
```python
#!/usr/bin/env python
print('hello, world!')
```
```shell
#!/bin/sh
echo "hello, world!"
```

**Review by spenserblack — COMMENTED** [review]
Thank you! Had a question about the interpreter entry.

File: gengo/languages.yaml (hunk 1 above)
Comment (spsandwichman) [reply gengo/languages.yaml:456]: oh my bad, from how the docs were worded I thought the interpreters field was for just associated programs, so i put the odin compiler. That can be taken out, then

Comment (spenserblack) [reply gengo/languages.yaml]: Thanks for the feedback! Perhaps I should make the docs more specific :thinking:

**Review by spenserblack — APPROVED** [review]

**Comment by spenserblack** [comment]
@all-contributors add @spsandwichman for data

**Pull request merged**